packages/next/next-server/server/config-utils.ts

```diff
@@ -122,7 +27,13 @@ function reasonMessage(reason: CheckReasons) {
 
 export async function loadWebpackHook(phase: string, dir: string) {
   let useWebpack5 = false
-  const worker: any = new Worker(__filename, { enableWorkerThreads: false })
+  const worker: any = new Worker(
+    path.resolve(__dirname, './config-utils-worker.js'),
+    {
+      enableWorkerThreads: true,
```

Comment:
I don't believe we can use worker threads here, IIRC it breaks Sharp cc @ijjk

Reply:
I can switch that back. It doesn't matter that much for performance.